official-antistasi-community/A3-Antistasi
# Adjust spawn vehicle velocity for spawnVehicle

## What type of PR is this.
* Bug
* Change
3. [x] Enhancement

### What have you changed and why?
Information:
* spawnVehicle now grabs the aircraft spawn speed from it's config stall speed.
* Tabs are converted to spaces.

### Please specify which Issue this PR Resolves.
closes https://github.com/official-antistasi-community/A3-Antistasi/issues/1864

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
* No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
// Spawn LSV in nearby "safe" position
```sqf
[getPos player, 0, "B_T_LSV_01_armed_F", resistance] call A3A_fnc_spawnVehicle params ["_vehicle", "_crew", "_group"];
```

// Spawn Helicopter at default height
```sqf
private _vehicle = [getPos player, 69, "B_Heli_Transport_01_F", group player] call A3A_fnc_spawnVehicle select 0;
_vehicle enableSimulation false; // For inspecting spawn.
```

// Spawn Helicopter above default height
```sqf
private _vehicle = [getPos player vectorAdd [0,0,420], 69, "B_Heli_Transport_01_F", group player] call A3A_fnc_spawnVehicle select 0;
_vehicle enableSimulation false; // For inspecting spawn.
```

// Spawn Helicopter bellow default height
```sqf
private _vehicle = [getPos player vectorAdd [0,0,50], 69, "B_Heli_Transport_01_F", group player, true] call A3A_fnc_spawnVehicle select 0;
_vehicle enableSimulation false; // For inspecting spawn.
```

// Release to check that helicopter RPM is stable.
```sqf
cursorObject enableSimulation true;
```

********************************************************
Notes:
The behaviour may need to be adjusted depending on peoples thoughts.
